cover.sh

```diff
@@ -11,13 +11,12 @@ echo "mode: count" > coverage.out
 # Initialize error tracking
 ERROR=""
 
-declare -a packages=('' \
-    'cmd' \
+declare -a packages=('cmd' \
```

Comment:
Since there are no `*.go` files now, Coveralls will complain if the `''` is left in.  Moved the other to be sorted.